plugins/modules/net_tools/nmcli.py

```diff
@@ -817,6 +817,10 @@ def connection_options(self, detect_change=False):
                 'bridge-port.hairpin-mode': self.hairpin,
                 'bridge-port.priority': self.slavepriority,
             })
+        elif self.type == 'team-slave':
```

Comment:
Could you please add tests for this change?

Reply:
@Akasurde I don't have much experience with test. Can you point me to any of the tests in tests/unit/plugins/modules/net_tools/test_nmcli.py similar to what you would implement so that I can take them as a reference please?

Reply:
Yes, sure. I will check once and get back to you.

Reply:
@Akasurde I tried to implement some tests. They work locally but failed on the CI. Some help will be very welcome.

Reply:
@omula What's the exact command you're running to run unit tests locally?
Also if you haven't already please take a look at: https://docs.ansible.com/ansible/devel/dev_guide/testing_units.html#running-tests

Reply:
This is the command I am using.
`ansible-test units --venv tests/unit/plugins/modules/net_tools/test_nmcli.py`

and also tried with:
`ansible-test units --docker  --python 2.7 tests/unit/plugins/modules/net_tools/test_nmcli.py`

Reply:
Executing `ansible-test units --docker -v nmcli` using this PR branch and latest source from devel passes for me as well.
It's odd...

Reply:
I was about to say it's odd indeed, when I remembered that in CI, the branch is rebased against `main` before the tests are run. After rebasing your branch against the latest `main`, unit tests now also fail for me. So try rebasing :)

Reply:
Thanks for the tip @felixfontein